src/applications/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1420,7 +1421,13 @@ export function getNextEvents(appeal) {
                   Your hearing will be transcribed and added to your appeal
                   file. The judge won’t make a decision about your appeal at the
                   hearing.{' '}
-                  <a href="/disability/file-an-appeal/board-of-veterans-appeals/">
+                  <a
+                    href={
+                      appealType === APPEAL_TYPES.appeal
+                        ? '/decision-reviews/board-appeal/veterans-law-judge-hearing/'
+                        : '/disability/file-an-appeal/board-of-veterans-appeals/'
```

Comment:
Since it's used in a couple places, it might make sense to move that value into a variable outside of the JSX.